packages/react-reconciler/src/ReactFiberClassUpdateQueue.new.js

```diff
@@ -108,6 +108,7 @@ import {
   ShouldCapture,
   DidCapture,
 } from './ReactFiberFlags';
+import getComponentNameFromFiber from 'react-reconciler/src/getComponentNameFromFiber';
```

Comment:
A relative import should work here.
```suggestion
import getComponentNameFromFiber from './getComponentNameFromFiber';
```